common/scripts/ci/ci_deploy_and_testcafe.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+#
+# This script builds and releases just the proxy to heroku. The proxy can be configured to work against any
```

Comment:
The comment is probably not right, please update it to reflect what this thing really does.